### PR TITLE
CircleCI Documentation

### PR DESCRIPTION
**Pull request by levlaz**
I have updated the documentation to include instructions on how to test atom on Ubuntu Linux using CircleCI. 

Also included is a sample circle.yml file and a link to a build using the same (wrap-guide) project as the other examples.

### CODE REVIEW

**Comment by thomasjo** [comment]
Thanks for taking the time to do this :bow: 

Personally I think we should stick with Travis CI, since that's we use for testing core Atom on Linux. I'd rather not add another CI provider to the mix if we don't have to. /cc @atom/feedback

**Comment by levlaz** [comment]
@thomasjo Thanks for the feedback! 

I certainly don't want to create any additional complexity, but I figured since CircleCI is now an ["official" GitHub integration](https://github.com/integrations/circle-ci ) it would be worth including in these docs. In addition CircleCI provides 3 free containers for all open source projects, so if developers have complex atom packages they can speed up their build times by running in parallel or get better throughput by running builds concurrently. 

In any case, there is nothing special or CircleCI specific about this setup so it seems to me like the steps can be transferred to travis if you decide to not include these docs. Let me know and I would be glad to write a "Testing on Linux" with Travis doc. 

Thank you all for your hard work, Atom is truly amazing. <3

File: README.md [hunk 1]
```diff
@@ -26,6 +27,13 @@ Templates for building your Atom package and running its specs:
 * :boom: Your package will now build and run its specs; you can see an example
   of a configured package [here](https://ci.appveyor.com/project/kevinsawicki/wrap-guide)
 
+### CircleCI
+
+* Sign up for an account on [CircleCI](https://circleci.com)
+* Add a new project
```
Comment (kevinsawicki) [inline README.md]: Is there a help guide to possibly link to for this action?

Comment (levlaz) [inline README.md]: @kevinsawicki Would something like this work? https://circleci.com/docs/getting-started

**Comment by mnquintana** [comment]
I think adding docs for Circle CI is a good idea, but I think it should be in addition to, not instead of, Travis docs for Linux (since we only use and support Travis for bundled packages). What do you think about adding Travis docs for Linux too?

File: README.md (hunk 1 above)
Comment (kevinsawicki) [inline README.md]: :+1: Yup, that would be great

**Comment by kevinsawicki** [comment]
> What do you think about adding Travis docs for Linux too?

Yeah, I'd like to have this too but I'm okay with it in a subsequent pull request.

File: README.md (hunk 1 above)
Comment (levlaz) [inline README.md]: @kevinsawicki I added the link to the documentation to the PR :)

**Comment by levlaz** [comment]
@mnquintana @kevinsawicki I would be glad to write up some Travis docs for Linux in a new PR. I do not believe they install X by default in their images so it will be a bit more work, but otherwise I do not see any foreseeable challenges.

**Comment by kevinsawicki** [comment]
Thanks for this :+1:

**Pull request merged**